stable/azuredisk-csi-driver/values.yaml

```diff
@@ -4,31 +4,31 @@ image:
     tag: v0.5.0
     pullPolicy: Always
   csiProvisioner:
-    repository: mcr.microsoft.com/oss/kubernetes-csi/csi-provisioner
+    repository: quay.io/k8scsi/csi-provisioner
     tag: v1.4.0
     pullPolicy: Always
   csiAttacher:
-    repository: mcr.microsoft.com/oss/kubernetes-csi/csi-attacher
+    repository: quay.io/k8scsi/csi-attacher
     tag: v1.2.0
     pullPolicy: Always
   clusterDriverRegistrar:
-    repository: mcr.microsoft.com/oss/kubernetes-csi/csi-cluster-driver-registrar
+    repository: quay.io/k8scsi/csi-cluster-driver-registrar
     tag: v1.0.1
     pullPolicy: Always
   csiSnapshotter:
-    repository: mcr.microsoft.com/oss/kubernetes-csi/csi-snapshotter
+    repository: quay.io/k8scsi/csi-snapshotter
     tag: v1.1.0
```

Comment:
v2.0.1 got released, but maybe something for the next change

Reply:
look at https://github.com/kubernetes-sigs/azuredisk-csi-driver for the next upgrade